HEROKU.md

```diff
@@ -119,7 +119,7 @@ If you need to check that the tables exist, you can connect to Postgres using so
 * `Select * from users;` or `Select * from reports;` will show you their contents
 * `\q` quits the psql terminal
 
-If needed, you can manually run the table creation script: `Heroku run php lib/db_create_tables.php`
+If needed, you can manually run the table creation script: `heroku run composer dbsetup`
```

Comment:
Switching to using composer scripts. so nice.

Composer is available on Heroku, no need to use `php composer.phar`